docs/getting-started/structure.mdx

```diff
@@ -0,0 +1,131 @@
+# Structure
+
+Widgetbook is flexible and powerful enough that fits into your projects structure.
+
+The structure of your components can be divided into two parts:
+
+1- The folder structure in your project, where you place files
+
+2- The navigation that will be created in Widgetbook
+
+Let's take a look at an typical Flutter example.
+In this case, you may add a `components` or `widgets` folder to your projects where you can manage your Ui catalogs structure.
+
+```bash
+your_flutter_project/
+├─ android/
+├─ ios/
+├─ lib/
+│ ├─ main.dart
+│ ├─ src/
+│ │ ├─ components/
+│ │ ├────── containers.dart
+│ │ ├────── buttons.dart
+│ │ ├─ models/
+│ │ └─ services/
+│ └─ ...
+├─ assets/
+│ ├─ images/
+│ ├─ fonts/
+│ └─ ...
+├─ test/
+├─ pubspec.yaml
+└─ ...
+```
+
+You may adjust based on your team and project's preferences.
+
+However, placing files in different folders doesn't necessarily reflect the navigation in Widgetbook.
+
+## Navigation structure
+
+The navigation structure in widgetbook is depending on which approach you select.
+
+<Tabs
+  groupId="approach"
+  values={[
+    { label: 'Generator', value: 'generator' },
+    { label: 'Manual', value: 'manual' },
+  ]}
+>
+  <TabItem value="generator">
+   The generator creates the navigation tree based on the file location of the component that has been specified for the `type` parameter of the `@WidgetbookUseCase` annotation. 
+   Therefore, the navigation structure of the Widgetbook resembles the structure of your application or package.
+
+    <Warning>
+        As of now, it's not possible to configure a different navigation structure using the generator.
+        The Widgetbook team aims to provide a solution in the future.
+    </Warning>
+
+  </TabItem>
+
+  <TabItem value="manual">
+    While you can directly add `WidgetbookComponent` to `directories` project on `Widgetbook` constructors, it will not be scalable and easy access as project grows.
+
+The alternatives are:
+
+1- WidgetbookCategory
+2- ¬WidgetbookFolder
```

Comment:
This looks odd in the preview 

<img width="420" alt="image" src="https://user-images.githubusercontent.com/16862002/233370741-699bdb84-e179-4bb8-a6f6-645ca768e43d.png">

Reply:
fixed